lib/components/split-pane.js

```diff
@@ -116,7 +115,7 @@ export default class SplitPane extends PureComponent {
       sizes = new Array(children.length).fill(1 / children.length);
     }
     return (
-      <div className={css('panes', `panes_${direction}`)}>
+      <div className={`splitpane_panes splitpane_panes_${direction}`}>
```

Comment:
`splitpane_${direction}`

Reply:
As I said about the other classnames, I didn't make it up. It's what has to be used to keep consistency and backwards compatibility.